Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/BukkitMain.java

```diff
@@ -739,8 +741,22 @@ public final ChunkGenerator getDefaultWorldGenerator(@NotNull final String world
             SetupObject setup = new SetupObject();
             setup.plotManager = manager;
             setup.setupGenerator = worldConfig.getString("generator.init", manager);
-            setup.type = worldConfig.getInt("generator.type");
-            setup.terrain = worldConfig.getInt("generator.terrain");
+            String type = worldConfig.getString("generator.type");
+            if (type == null) {
```

Comment:
This exact snipper (parsing type, and terrain) exists in two different places, with identical parsing logic. It might be a good idea to move this into the enum types themselves?

Reply:
Yeah, I'll take a look at that again